src/authorization.ts

```diff
@@ -19,6 +19,11 @@ export interface AuthorizationBundleRequest {
     requests: BatchReadWriteRequest[];
 }
 
+export interface getAllowedResourceTypesForOperationRequest {

```

Comment:
Thoughts: At a minimum we need to capitalize the 'g' in get
```suggestion
export interface ResourceTypesForOperationRequest {

```

Reply:
updated to `AllowedResourceTypesForOperationRequest`